src/foam/nanos/crunch/ui/CapabilityInterceptView.js

```diff
@@ -0,0 +1,77 @@
+foam.CLASS({
+  package: 'foam.nanos.crunch.ui',
+  name: 'CapabilityInterceptView',
+  extends: 'foam.u2.Element',
+
+  implements: [ 'foam.mlang.Expressions' ],
+
+  requires: [
+    'foam.nanos.crunch.Capability'
+  ],
+
+  imports: [
+    'stack'
+  ],
+
+  properties: [
+    { name: 'data' },
+    { name: 'capabilityOptions', class: 'StringArray' }
+  ],
+
+  css: `
+    ^ {
+      width: 1024px;
+      margin: auto;
+    }
+    ^action-container {
+      display: flex;
+      justify-content: space-between;
+      margin: 8px 0;
+    }
+    ^action-container > div > div > * + * {
+      margin-left: 8px;
+    }
+    ^detail-container {
+      overflow-x: scroll;
+    }
+  `,
+
+  methods: [
+    function initE() {
+      this
+        .addClass(this.myClass())
+
+        // Container for the actions
```

Comment:
We don't need the container since you're not including another group of buttons on the right, like the place this was copied from. Might as well change lines 44-54 to the following:

```suggestion
        .startContext({ data: this })
          .tag(this.CANCEL, { buttonStyle: 'SECONDARY' })
        .endContext()
```